src/vs/workbench/contrib/terminal/browser/terminalTabsList.ts

```diff
@@ -707,10 +708,27 @@ class TerminalTabsDragAndDrop extends Disposable implements IListDragAndDrop<ITe
 		}
 
 		let focused = false;
+		// Retrieve the currently active instance asynchronously
+		let activeInstancePromise = this._terminalService.activeInstance;
+		let setActiveInstance = async (instance: ITerminalInstance) => {
+			const activeInstance = await activeInstancePromise; // Ensure the promise is resolved before comparison.
```

Comment:
`this._terminalService.activeInstance` is syunc, so this await doesn't seem to do anything.